src/Configuration/RectorConfigBuilder.php

```diff
@@ -469,6 +478,82 @@ public function withPhpSets(
         return $this;
     }
 
+    // suitable for PHP 7.4 and lower, before named args
+    public function withPhp53Sets(): self
+    {
+        Notifier::notifyNotSuitableMethodForPHP80(__METHOD__, 'withPhpSets');
```

Comment:
The warning seems shown multiple times due to **parallel**:

```                                                                           
 [WARNING] The "Rector\Configuration\RectorConfigBuilder::withPhp53Sets()"      
           method is suitable for PHP 7.4 and lower. Use "withPhpSets()" method 
           instead.                                                             
                                                                                

  0/99 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0%
                                                                                
 [WARNING] The "Rector\Configuration\RectorConfigBuilder::withPhp53Sets()"      

           method is suitable for PHP 7.4 and lower. Use "withPhpSets()" method 
           instead.                                                      
                                                                                
 [WARNING] The "Rector\Configuration\RectorConfigBuilder::withPhp53Sets()"      
           method is suitable for PHP 7.4 and lower. Use "withPhpSets()" method 
           instead.                                                             
                                                                              
                                                                                
 [WARNING] The "Rector\Configuration\RectorConfigBuilder::withPhp53Sets()"      
           method is suitable for PHP 7.4 and lower. Use "withPhpSets()" method 
           instead.                                                        
```

that can't be tweak here, only on `ApplicationFileProcessor` level, the single warning can be shown...

Also, I am thinking if github workflow uses:

```
      matrix:
        php-versions: ['7.4, '8.0']
```

that may not suit, so probably suggest to use `withSets()`...